### PR TITLE
Adding gMSA documentation

### PR DESCRIPTION
**Pull request by msjaydeep**
Adding documentation for gMSA.

### CODE REVIEW

File: biztalk/core/gmsa-config.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ms.reviewer: 
+ms.suite:
+ms.tgt_pltfrm:
+ms.assetid: 
+ms.custom: 
+---
+
+# Using Group Managed Service Account for BizTalk Server Features
+
+BizTalk 2020 Supports Group Managed Service Account. Detailed information about gMSA can be found [here](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) .
```
Comment (Elvis-Shi) [inline biztalk/core/gmsa-config.md]: remove /en-us/ from the link? so it will redirect to current language user is using.

**Review by pariksg — COMMENTED** [review]
Pictures are not showing up on the page.

File: biztalk/core/gmsa-config.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+| Rest API | Yes |
+| BizTalk TMS | Yes |
+
+New installations of BizTalk may be configured with gMSA(where ever supported) by running Microsoft BizTalk Server Configuration. 
+
+> [!NOTE]
+> gMSA is not available in basic configuration.
+
+The features where gMSA is supported shows a check box which when checked password gets disabled. The user name should be set to appropriate gMSA account.
+
```
Comment (pariksg) [inline biztalk/core/gmsa-config.md:48]: images do not have $ in the gmsa account, this is misleading.

**Review by pariksg — DISMISSED** [review]

File: biztalk/core/gmsa-config.md (hunk 2 above)
Comment (msjaydeep) [reply biztalk/core/gmsa-config.md:48]: done

**Review by pariksg — APPROVED** [review]

**Pull request merged**